### ISSUE TITLE
Markdown rendering issue

### ISSUE THREAD

**Issue by rajurh** [norwoodj/helm-docs#68]
As per the documentation, it says below we should be able to generate readme with description from the below example

```controller:
  livenessProbe:
    httpGet:
      # -- This is the liveness check endpoint
      path: /healthz
      port: http
```

When I generate using docker image ```docker run --rm --volume "$(pwd):/helm-docs" -u $(id -u) jnorwood/helm-docs:latest --dry-run```

ReadME generated but the description is not populated as described in the documentation
![image](https://user-images.githubusercontent.com/15551296/96695892-a8c7bc00-13e6-11eb-82e9-f89533b5a542.png)

Documentation says
![image](https://user-images.githubusercontent.com/15551296/96696080-dca2e180-13e6-11eb-9b84-05ace0173d6a.png)

**Comment by norwoodj** [comment]
I believe you have an old version of the image cached locally:
```
$ docker run --rm --volume "$(pwd):/helm-docs" -u $(id -u) jnorwood/helm-docs:latest --dry-run
....
## Values

| Key | Type | Default | Description |
|-----|------|---------|-------------|
| livenessProbe.httpGet.path | string | `"/healthz"` | This is the liveness check endpoint |
| livenessProbe.httpGet.port | string | `"http"` |  |

----------------------------------------------
Autogenerated from chart metadata using [helm-docs v1.4.0](https://github.com/norwoodj/helm-docs/releases/v1.4.0)
```

You should see a footer with the version of helm-docs that was used to generate the markdown, assuming you have helm-docs >= v1.2.0

I'd recommend you try either:
```
docker pull jnorwood/helm-docs:latest
# or
docker run --rm --volume "$(pwd):/helm-docs" -u $(id -u) jnorwood/helm-docs:v1.4.0 --dry-run
```

**Comment by rajurh** [comment]
> docker run --rm --volume "$(pwd):/helm-docs" -u $(id -u) jnorwood/helm-docs:v1.4.0 --dry-run

I was trying with the latest and just to reconfirm I cleared cached images and with v1.4.0 issue still exists
![image](https://user-images.githubusercontent.com/15551296/96785054-be6ecd00-144a-11eb-9b1c-d4b3aa1b221f.png)

**Comment by norwoodj** [comment]
Hmmm strange. Would you mind creating a zip or tar archive of the chart that you're running this on? It would help me debug if I have all your files exactly as you have them.

**Comment by rajurh** [comment]
I have created sample one with a value file mentioning comments 
[test.zip](https://github.com/norwoodj/helm-docs/files/5420507/test.zip)

Sample output
![image](https://user-images.githubusercontent.com/15551296/96832716-b12dfe80-149b-11eb-878d-110cb2172b3e.png)

**Comment by norwoodj** [comment]
Apologies that I haven't had time to look at this. I haven't been able to reproduce this issue with my own charts, but I still need to look at the zip archive you've sent me. Sorry for the delay, I promise I'll look soon

**Comment by norwoodj** [comment]
@rajurh I'm again really sorry how long it has taken me to look at this. The bug, as it turns out is within the yaml parsing library the tool uses. The values.yaml file you have provided me obviously came from a windows machine, as it uses `\r\n` for line endings.

For whatever reason the underlying yaml library has trouble with such line endings. I have fixed the issue finally and will release v1.5.0 shortly with the fix.

**Comment by rajurh** [comment]
Thanks, @norwoodj  issue is fixed
